index.js

```diff
@@ -12,16 +12,15 @@
  │   See the License for the specific language governing permissions and       │
  │   limitations under the License.                                            │
  \*───────────────────────────────────────────────────────────────────────────*/
-'use strict';
```

Comment:
@skratchdot : I tested it on v6.10.2 & v7.9.0 its working ... will check on v4 ranges and update soon